### DISCUSSION TITLE
Prior Austenite Grain Reconstruction

### DISCUSSION THREAD

**Discussion by saurabhp92** [mtex-toolbox/mtex#735] [Ask Anything] [answered]
Dear all,
I am trying to reconstruct PAG for my martensite EBSD data. The code ran successfully for many cases but showing following error for one case.

>>Error using convhull
>>Error computing the convex hull. Not enough unique points specified.

>>Error in spatialDecomposition>calcBoundary (line 96)
      k = convhull(x,y);

>>Error in spatialDecomposition (line 26)
  dummyCoordinates = calcBoundary(X,unitCell,varargin{:});

>>Error in EBSD/calcGrains (line 26)
[V,F,I_FD] = spatialDecomposition([ebsd.prop.x(:), ebsd.prop.y(:)],ebsd.unitCell,varargin{:});

>>Error in PAG_MTEX (line 36)
[grains,ebsd.grainId] = calcGrains(ebsd('Ferrite'),'angle',3*degree);

Here is the first Block of my code

%% Specify File Names
 
% path to files
pname = 'C:\Users\Saurabh\Desktop\EBSD\Experiment\Attempt_2\5kA\Initial_5kA\PAG';
 
% which files to be imported
fname = [pname '\cropped.ang'];
 
%% Import the Data
 
% create an EBSD variable containing the data
ebsd = EBSD.load(fname,CS,'interface','ang',...
  'convertEuler2SpatialReferenceFrame');
 
**_[grains,ebsd.grainId] = calcGrains(ebsd('indexed'),'angle',3*degree); Showing error for this line_**

Please help with this issue

Thanks and Regards 
Saurabh Pawar

**Comment by kilir** [comment, marked as answer]
Hi,
what is the output of `ebsd` , repsectively `ebsd('Ferrite')`?
Cheers,
Rüdiger

**Reply by saurabhp92** [reply]
yes.

**Reply by saurabhp92** [reply]
it worked 

thanks alot though
